flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -72,6 +72,13 @@
 			.defaultValue(16)
 			.withDeprecatedKeys("job-manager.max-attempts-history-size");
 
+	/**
+	 * The maximum number of prior execution attempts kept in history.
+	 */
+	public static final ConfigOption<String> EXECUTION_FAILOVER_STRATEGY =
+		key("jobmanager.execution.failover-strategy")
+			.defaultValue("full");
```

Comment:
I don't think we have done that yet, no...

There is a PR to extend the config options to include a description (and generate docs), it could be an extension there.